src/test/resources/feature/2-mittente/4_visualizzazioneNotifiche/VisualizzaNotifica.feature

```diff
@@ -16,7 +16,7 @@ Feature: Mittente visualizza il dettaglio di una notifica con documenti allegati
     And Logout da portale mittente
 
   @CheckNotificaConDocumentiAllegati
-  Scenario: PN-10390 - Mittente visualizza dettaglio notifica con attestazione opponibile a terzi notifica presa in carico
+  Scenario: [TA-FE VISUALLIZZAZIONE DETTAGLI DI NOTIFICA] - Mittente visualizza dettaglio notifica con attestazione opponibile a terzi notifica presa in carico
```

Comment:
this title should be different from other and unique, fix VISUALLIZZAZIONE in VISUALIZZAZIONE